harness/determined/common/api/authentication.py

```diff
@@ -151,6 +145,63 @@ def do_login(
     return token
 
 
+class LogoutAuthentication(Authentication):
+    """
+    An api-compatible Authentication object that is basically exactly a UserTokenPair.
+
+    TODO(XXX): delete Authentication class and write a function that returns a UsernameTokenPair
```

Comment:
If we approve this PR, I will replace this `XXX` with a real ticket number before landing.

Reply:
done